### ISSUE TITLE
Validating messages

### ISSUE THREAD

**Issue by brunocasarotti** [ACken2/bip322#7]
Hi, I am trying to validate a message signed from this address 3Agx7m86mJgVbLZP3Wk1qjYkzv6gGemz9X (p2sh). The verifySignature method states that "Verify a BIP-322 signature from P2WPKH, P2SH-P2WPKH, and single-key-spend P2TR address."

Should I be able to verify a signature of the following message?
message=`48656c6c6f20426974636f696e2034352e3133302e3130352e313436`
address=`3Agx7m86mJgVbLZP3Wk1qjYkzv6gGemz9X`
signature=`JDkLNaM8vWoobA34PGQE9FIZaLF7peRh4r7DOqOHls1cP1DPwR3Hcy26+zk6yRb0qtJRHEdUflVxkScbwsOCSMw=`

It failse with the error:
```
{
    "errorType": "RangeError",
    "errorMessage": "The value of \"offset\" is out of range. It must be >= 0 and <= 63. Received 300",
    "code": "ERR_OUT_OF_RANGE",
    "stack": [
        "RangeError [ERR_OUT_OF_RANGE]: The value of \"offset\" is out of range. It must be >= 0 and <= 63. Received 300",
        "    at new NodeError (node:internal/errors:405:5)",
        "    at boundsError (node:internal/buffer:88:9)",
        "    at Buffer.readUInt8 (node:internal/buffer:254:5)",
        "    at Object.decode (/var/task/node_modules/bip174/src/lib/converter/varint.js:43:24)",
        "    at readVarInt (/var/task/node_modules/bitcoinjs-lib/src/psbt.js:1492:24)",
        "    at readVarSlice (/var/task/node_modules/bitcoinjs-lib/src/psbt.js:1497:22)",
        "    at readVector (/var/task/node_modules/bitcoinjs-lib/src/psbt.js:1502:49)",
        "    at scriptWitnessToWitnessStack (/var/task/node_modules/bitcoinjs-lib/src/psbt.js:1505:10)",
        "    at /var/task/node_modules/bitcoinjs-lib/src/psbt.js:1557:29",
        "    at Array.forEach (<anonymous>)"
    ]
}
```

**Comment by ACken2** [comment]
Thank you for reporting the issue.

I couldn't reproduce your issue with version 1.1.0. Could you please check and confirm which version you're currently using?

However, I did discover an issue where version 1.1.0 incorrectly fails to validate your signature. This doesn't match the result obtained from https://www.verifybitcoinmessage.com/. It seems to be related to the flags setting during BIP-137 signature validation.

**Comment by brunocasarotti** [comment]
Yes I was using version 1.1.0
EDIT:
I may have put the wrong exception though, I just tested it again and just got a FALSE as result, which as you said, doesn't match with the verifybitcoinmessage website

**Comment by ACken2** [comment]
This issue has been fixed in v1.1.1 release.

The following code should work now:
```
const { Verifier } = require('bip322-js');
const address = "3Agx7m86mJgVbLZP3Wk1qjYkzv6gGemz9X";
const message = "48656c6c6f20426974636f696e2034352e3133302e3130352e313436";
const signature = "JDkLNaM8vWoobA34PGQE9FIZaLF7peRh4r7DOqOHls1cP1DPwR3Hcy26+zk6yRb0qtJRHEdUflVxkScbwsOCSMw=";

console.log(Verifier.verifySignature(address, message, signature)); // True
```
